docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -1,25 +1,375 @@
+
+Now you can safely apply the update or upgrade.
+
+.. warning::
+
+   Please remind, that |dir| and |sd| must always have the same version.
+   The version of the File Daemon may differ, see chapter about :ref:`backward compatibility <backward-compatibility>`.
+
+
+.. note::
+
+   When you change the repository and refresh it, it is likely to have a new signing key.
+   You will have to accept it.
+
+
+.. code-block:: shell-session
+   :caption: new gpg key detection on EL 8
+
+      Backup Archiving Recovery Open Sourced (EL_8)                                                                                                                                                               1.6 MB/s | 1.6 kB     00:00
+      Importing GPG key 0xC9FED482:
+      Userid     : "Bareos 21 Signing Key <signing@bareos.com>"
+      Fingerprint: 91DA 1DC3 564A E20A 76C4 CA88 E019 57D6 C9FE D482
+      From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-bareos-release-21
+      Is this ok [y/N]: y
+
+
+.. code-block:: shell-session
+   :caption: new gpg key detection on SUSE
+
+      Forcing raw metadata refresh
+      New repository or package signing key received:
+      Repository:       bareos
+      Key Fingerprint:  91DA 1DC3 564A E20A 76C4 CA88 E019 57D6 C9FE D482
+      Key Name:         Bareos 21 Signing Key <signing@bareos.com>
+      Key Algorithm:    RSA 4096
+      Key Created:      Mon Dec 20 10:04:50 2021
+      Key Expires:      (does not expire)
+      Rpm Name:         gpg-pubkey-c9fed482-61c05542
+
+         Note: Signing data enables the recipient to verify that no modifications occurred after the data
+         were signed. Accepting data with no, wrong or unknown signature can lead to a corrupted system
+         and in extreme cases even to a system compromise.
+
+         Note: A GPG pubkey is clearly identified by its fingerprint. Do not rely on the key's name. If
+         you are not sure whether the presented key is authentic, ask the repository provider or check
+         their web site. Many providers maintain a web page showing the fingerprints of the GPG keys they
+         are using.
+
+      Do you want to reject the key, trust temporarily, or trust always? [r/t/a/?] (r): a```
+
+
+
+.. _bareos-update_from_org_to_com:
+
+Updating from community to subscription binaries
+------------------------------------------------
+
+To update the installed packages to Bareos Subscription binaries, you will have to point to the subscription repositories located at https://download.bareos.com/.
+
+Once you received your `download.bareos.com` portal credentials, you can refer to the following section :ref:`section-AddSoftwareRepository` for complete instructions how-to use the :file:`add_bareos_repositories.sh` helper.
+
+Choose the same operating system and Bareos major version you are already using. You will then benefit the last bug fixes and enhancements, without having to do a major upgrade immediately.
```

Comment:
```suggestion
Choose the same operating system and Bareos major version you are already using.
```
Strictly speaking, this is not true anymore after we have `current` on the bareos.org server.

Reply:
I agree, but then maybe we will speak about upgrading to current for community. Here the chapter can be applied to both and for d.b.c it will still be the case.